content/en/examples/deployments/deployment-with-low-priorityclass.yaml

```
@@ -0,0 +1,32 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: placeholder-deployment
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: placeholder
+  template:
+    metadata:
+      labels:
+        app: placeholder
+    spec:
+      priorityClassName: low-priority
+      affinity:
+        podAntiAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
```

```suggestion
          preferredDuringSchedulingIgnoredDuringExecution:
```

(IMO) we shouldn't trigger node creation just to make the placeholder Pods.